Tests/PerformanceTestingTests/ArrayTests.swift

```diff
@@ -13,7 +13,7 @@ class ArrayTests: PerformanceTestCase {
     // MARK: Helper functions.
 
     // Constructs an array of size `n` with linearly increasing elements.
-    func constructSizeNArray(size n: Int) -> [Int] {
+    func constructArray(size n: Int) -> [Int] {
```

Comment:
I like this name better than the prev, forsure.

One rule I've been following throughout this codebase (as best I am able) is:

- `make`: returns new value with no side effects
- `create`: returns new value while storing it internally (side effect)
- `add` / `insert` / etc.: creates new value, storing it internally, not returning it